src/LightGraphs.jl

```diff
@@ -10,17 +10,17 @@ using Clustering: kmeans
 import GraphMatrices: CombinatorialAdjacency
 using JuMP
 
-import Base: write, ==, <, *, isless, issubset, complement, union, intersect,
+import Base: write, ==, <, *, isless, issubset, union, intersect,
             reverse, reverse!, blkdiag, getindex, setindex!, show, print, copy, in,
-            sum, size, sparse, eltype, length, ndims, issym, transpose,
-            ctranspose, join, start, next, done, eltype, get
+            sum, size, sparse, eltype, length, ndims, issymmetric, transpose,
+            ctranspose, join, start, next, done, eltype, get, length
 
 
 # core
-export SimpleGraph, Edge, Graph, DiGraph, vertices, edges, src, dst,
+export SimpleGraph, Edge, EdgeIter, Graph, DiGraph, vertices, edges, src, dst,
```

Comment:
EdgeIter should remain unexported. It's an internal construct we don't want to expose.